signer/src/keypair.rs

```diff
@@ -93,6 +93,13 @@ impl Keypair {
     pub fn to_hex(&self) -> String {
         hex::encode(self.to_bytes())
     }
+
+    /// Performs scalar multiplication with the secret key
```

Comment:
Opinion: I would inline it in the OpenMina codebase, see https://github.com/o1-labs/proof-systems/pull/2755. Curious to see the other team members opinions.